src/main/java/frc/robot/Path.java

```diff
@@ -16,7 +14,7 @@ public Path(String PathName){
         pathName = PathName;
     }    
 
-    public ArrayList returnLeftList() throws IOException{
+    public ArrayList<String> returnLeftList() throws IOException{
```

Comment:
Change the ArrayList to doubles, that way you don't have to toString when you are in the path's run command.